include/usearch/index_dense.hpp

```diff
@@ -748,11 +750,10 @@ class index_dense_gt {
         unique_lock_t lookup_lock(slot_lookup_mutex_);
 
         std::unique_lock<std::mutex> free_lock(free_keys_mutex_);
+        // storage_ cleared by typed_ todo:: is this confusing?
         typed_->clear();
```

Comment:
`storage_` is reset by `typed_.reset()`;
nothing bad will happen here if I do it again, but this seemed clearer